erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -319,8 +319,11 @@ def check_credit_limit(self):
 			if d.party_type=="Customer" and d.party and flt(d.debit) > 0]))
 		if customers:
 			from erpnext.selling.doctype.customer.customer import check_credit_limit
+
 			for customer in customers:
-				check_credit_limit(customer, self.company)
+				extra_amount = sum([flt(d.debit) for d in self.get("accounts")
+					if d.party_type=="Customer" and d.party==customer and flt(d.debit) > 0])
+				check_credit_limit(customer, self.company, extra_amount=extra_amount)
```

Comment:
Instead of giving it `extra_amount` a better way to handle this is to do the credit limit check after posting the GLs. That would be a more consistent and safer way to do credit limit checks.

Revert and move the `check_credit_limit` after `make_gl_entries` like this
<img width="638" height="356" alt="image" src="https://github.com/user-attachments/assets/ead2c2cf-116e-4168-9940-e8ad69d2ecdb" />